src/client/containers/Admin-Event-details/Admin-Event-modal-view.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { EventCardText } from '../../components/EventCardText/EventCardText';
+import Modal from '../../components/Modal/Modal';
+import { StudyGroupLogo } from '../../components/StudyGroupLogo/StudyGroupLogo';
+
+import Elasticlogo from './Elastic.logo.svg';
+import './Admin-event-modal.style.css';
+
+export default function EventModalView() {
+  return (
+    <div className="event-modal-view">
+      <Modal>
+        <div className="study-group-info">
+          <div className="study-group-logo">
+            <StudyGroupLogo path={Elasticlogo} />
+          </div>
+          <div className="card-text">
+            <EventCardText
+              title="Elastic - Week 46"
+              datetime="Wednesday | 11 Nov 20|10:00 CEST"
+              mentor="Maher"
+              link="www.hyf.org"
```

Comment:
You cannot hard-code these here since these depend on what your event actually contains. They should come through props.